frontend/src/container/Tracim.jsx

```diff
@@ -499,8 +498,8 @@ export class Tracim extends React.Component {
 
                 <Route
                   path={PAGE.WORKSPACE.RECENT_ACTIVITIES(':idws')}
-                  render={(routerProps) => (
-                    <WorkspaceRecentActivities workspaceId={routerProps.match.params.idws} />
+                  render={({ match }) => (
+                    <Redirect to={PAGE.WORKSPACE.DASHBOARD(match.params.idws)} />
```

Comment:
do we need to keep this route?
I guess to avoid dead link from user's browser history
I dunno

Reply:
I think we should take a habit of not breaking routes, though for this particular case, /recent-activities have not existed for long so we can remove it. It will require adapting a few cypress tests to remove it so they go to the dashboard instead.

What do you think?

Reply:
(Actually, I'd be more inclined to redirect /activity-feed, which has existed for longer)

Reply:
I guess we can keep it but I'd like some kind of deprecation comment

Reply:
```suggestion
                  render={({ match }) => (
                     // NOTE - RJ - 2021-03-29 - This redirection is there to avoid breaking old links to recent activities
                     // We may want to remove this redirection in the future. Beware to fix the related Cypress tests 
                    <Redirect to={PAGE.WORKSPACE.DASHBOARD(match.params.idws)} />
```